src/community/advanced-data-card.tsx

```diff
@@ -289,6 +281,7 @@ export const AdvancedDataCard = React.forwardRef<HTMLDivElement, AdvancedDataCar
                         maybe
                         className={classNames(styles.touchableContainer, {[styles.hoverEffect]: !!onPress})}
                         aria-label={ariaLabel}
+                        role="link"
```

Comment:
why role link?
`Touchable` renders as html `<button>` (role button) when used with `onPress` prop and as html `<a>` (role link) when used with `href`/`to` prop . If you want a link, I think the best approach is adding `href`/`to` support to `AdvancedDataCard` as we have in other cards

Reply:
Hello Abel,

This PR is still under review of accessibility team, we upload here to generate the preview link for them.
Most grateful for your contribution at this time :)

Reply:
the role='link' was to be able to adjust the accessibility semantics without changing any properties, but I implemented the href according to your suggestion